README.md

```diff
@@ -49,6 +49,10 @@ Mongoid::Migration.after_migrate = ->(output, name, direction) {
 
 ## Unreleased
 
+## 1.4.0
+_06/01/2021_
+* The hook `migrations_path` can be use when migration crash (#56)
+
```

Comment:
The hook is `after_migrate`, right? also could you please:
- Put the date for 1.3.0 release
- Move the "Compare master with" link up, and change it to compare with the last version (the goal is to quickly see unreleased changes)
- For 1.4.0 you can update the date to today I may have the time to release it after these fixes ;)